docs/vpcanalyzer_lint.md

```diff
@@ -0,0 +1,93 @@
+## vpcanalyzer lint
+
+provides a detailed report of potential issues in the given VPC configuration
+
+### Synopsis
+
+The following linters are supported:
+
+| Name                            | Description                                                                              |
+|---------------------------------|------------------------------------------------------------------------------------------|
+| **nacl-split-subnet**           | Network ACLs implying different connectivity for endpoints inside a subnet               | 
+| **sg-split-subnet**             | SGs implying different connectivity for endpoints inside a subnet                        |  
+| **subnet-cidr-overlap**         | Overlapping subnet address spaces                                                        |  
+| **nacl-unattached**             | Network ACL not applied to any resources                                                 |  
+| **sg-unattached**               | SG not applied to any resources                                                          |  
+| **sg-rule-cidr-out-of-range**   | rules of security groups that reference CIDRs not in the relevant VPC address range      |
```

Comment:
```suggestion
| **sg-rule-cidr-out-of-range**   | Security-group rules referencing CIDRs outside of the VPC address space    |
```

Reply:
I copied the text here from the linters descriptions - and will copy back every change we end doing from README to the description in the code

Reply:
done (with "Security group")